src/index.html

```diff
@@ -15,8 +15,94 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.item"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a>
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item firstItem">
+            <a
+              class="nav_link"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav_item">
```